charts/fulcio/values.schema.json

```diff
@@ -64,8 +64,14 @@
                 "ingress": {
                     "enabled": true,
                     "annotations": {},
-                    "path": "/",
-                    "tls": {}
+                    "hosts": [
+                      {
+                        "host": "fulcio.localhost",
+                        "path": "/"
+                      }
+                    ],
+                    "tls": [],
+                    "tlsEnabled": false
```

Comment:
Question: why do we need this flag? Wouldn’t it be enough with the array size as a flag?

Reply:
good point. removed.